docs/developer/release/release_utils.py

```diff
@@ -557,7 +557,7 @@ def get_announcement_email(version):
 
 Changes for Checker Framework version %s:
```

Comment:
Should this line be removed or changed? Otherwise the email will look like:
```
Changes for Checker Framework version 3.34.0:

Version 3.34.0 (May 2, 2023)
------------------------------

**User-visible changes:**
```

Reply:
I don't think it much matters.